### ISSUE TITLE
How to finetune a large number of scRNA datasets?

### ISSUE THREAD

**Issue by EddieLv** [vandijklab/cell2sentence#22]
Thanks for your amazing work!
I want to finetune a customized model based on a large number of scRNA datasets, for example, 1 million lung single-cells of different studys/batches, in order to perform better on lung cell type prediction.
I found that the test data (29k single-cells) will use about 60GB GPU memory, which is very big, so is there any way to reduce the usage of GPU memory for big data training? Or is there any better approaches for big data training?

**Comment by MKHarsha03** [comment]
Use quantization techniques to reduce the parameter precisions, and train using LoRA modules. Use gradient accumulation and small batch sizes. Keep playing around with different batch sizes until you find the optimal hyperparameters.
